net/ddns-scripts/files/usr/lib/ddns/update_ns1_com.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+# Derived from update_gandi_net.sh
+
+. /usr/share/libubox/jshn.sh
+
+add_answer() { # {"answers":[{"answer":["1.1.1.1"]}]}
+	json_add_array answers
+		json_add_object
+			json_add_array answer
+				json_add_string "" "$1"
+			json_close_array
+		json_close_object
+	json_close_array
+}
+
+local __ENDPOINT="https://api.nsone.net/v1"
+local __TTL=600
+local __RRTYPE
+local __URL
+local __STATUS
+
+[ -z "$username" ] && write_log 14 "Service section not configured correctly! Missing zone as 'username'"
+[ -z "$password" ] && write_log 14 "Service section not configured correctly! Missing API Key as 'password'"
+
+[ $use_ipv6 -ne 0 ] && __RRTYPE="AAAA" || __RRTYPE="A"
+
+# Construct JSON payload
+json_init
+add_answer "$__IP"
```

Comment:
Could you please remove this function.
I see no point in writing a sub-function here for two calls.